Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 resolver = "2"
-members = ["crates/proof-of-sql", "crates/proof-of-sql-parser"]
+members = ["crates/proof-of-sql", "crates/proof-of-sql-parser", "crates/proof-of-sql-utils"]
```

Comment:
@Bhavyajain21 The crate no longer exists so please remove it here as well

Reply:
On it

Reply:
Done! @iajoiner